.github/actions/setup-gcloud/action.yml

```diff
@@ -0,0 +1,33 @@
+name: Setup gcloud
```

Comment:
Note that this will need changes to the terraform, as this is keyless authentication which is based on the specific repo (animo/paradym)